packages/lib-classifier/src/plugins/drawingTools/models/marks/Mark/Mark.js

```diff
@@ -7,6 +7,7 @@ import AnnotationsStore from '@store/AnnotationsStore'
 
 const BaseMark = types.model('BaseMark', {
   id: types.identifier,
+  angle: types.maybe(types.number),
```

Comment:
I think the `BaseMark` model should only have properties that are actually common to all marks. This causes the point mark annotation to be submitted with `angle: undefined` in the classification. I think this could potentially cause confusion in the downstream analysis (someone will ask why points have undefined angles).